pkg/operator/controller/garden/components.go

```diff
@@ -163,3 +167,26 @@ func (r *Reconciler) newEtcd(
 		},
 	), nil
 }
+
+func (r *Reconciler) newKubeAPIServerService(log logr.Logger, garden *operatorv1alpha1.Garden) component.DeployWaiter {
+	var annotations map[string]string
+	if settings := garden.Spec.RuntimeCluster.Settings; settings != nil && settings.LoadBalancerServices != nil {
+		annotations = settings.LoadBalancerServices.Annotations
+	}
+
+	return kubeapiserverexposure.NewService(
```

Comment:
The [selector of the LB service](https://github.com/rfranzke/gardener/blob/217acd7a97a1af809f2012215a00d216213b2f60/pkg/operation/botanist/component/kubeapiserverexposure/kube_apiserver_service.go#L151) does not fit to the virtual garden, [because it points statically](https://github.com/rfranzke/gardener/blob/217acd7a97a1af809f2012215a00d216213b2f60/pkg/operation/botanist/component/kubeapiserverexposure/kube_apiserver_service.go#L206-L211) to
```
  selector:
    app: kubernetes
    role: apiserver
```

Reply:
There is no `kube-apiserver` deployment yet, hence this service does not select anything right now. In the future, `gardener-operator` will also deploy the `kube-apiserver` which will have exactly those labels. :)

Reply:
"The future" should be the next release, that we can still access the virtual garden 😅

Reply:
Not strictly necessary. If you deploy this to an existing landscape with a virtual garden then you have to migrate your existing deployment accordingly so that the labels match.